lib/widgets/compose_box.dart

```diff
@@ -205,44 +205,6 @@ class _ContentInput extends StatefulWidget {
 }
 
 class _ContentInputState extends State<_ContentInput> {
-  MentionAutocompleteView? _mentionAutocompleteView; // TODO different autocomplete view types
```

Comment:
In this version, `_ContentInputState` no longer carries any state, and so could be folded back into `_ContentInput` as a stateless widget.

(Not sure whether that'll remain true in a future version.)